docker/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM python:3.8-alpine
+
+ENV APP_NAME=MAGE
+ENV WORKDIR=/var/www/$APP_NAME \
+    DATABASE_ENGINE="" \
+    DATABASE_NAME="" DATABASE_HOST="" DATABASE_PORT="" DATABASE_USER="" DATABASE_PASSWORD="" \
+    ALLOWED_HOSTS="localhost" DEBUG=""  INTERNAL_IPS="" \
+    MEDIA_ROOT="" MEDIA_URL="" STATIC_ROOT="" STATIC_URL="" \
+    SECRET_KEY="your_own_here" AZURE_ACCOUNT_NAME="" AZURE_ACCOUNT_KEY="" AZURE_CONTAINER="" \
+    MAGE_ALLOW_MIGRATIONS=False
+
+ENV APP_DIR=$WORKDIR
```

Comment:
pas la peine de faire 3 ENV, je me moque des répétitions. (pas contre je n'aime pas avoir bcp d'instructions et donc de couches dans l'image résultante)

Reply:
Je comprends. Le soucis est surtout que 
ENV APP_NAME=MAGE
ENV WORKDIR=/var/www/$APP_NAME
ENV APP_DIR=$WORKDIR

Ne de donne pas le même résultat que 

ENV APP_NAME=MAGE \
  WORKDIR=/var/www/$APP_NAME \
  APP_DIR=$WORKDIR
  
Voir la doc
Environment variable substitution will use the same value for each variable throughout the entire instruction. In other words, in this example:

```
ENV abc=hello
ENV abc=bye def=$abc
ENV ghi=$abc
```
will result in def having a value of hello, not bye. However, ghi will have a value of bye because it is not part of the same instruction that set abc to bye.

source: https://docs.docker.com/engine/reference/builder/#environment-replacement

Reply:
Yep. ce que je veux dire par répétitions c'est : pq ne pas mettre `WORKDIR=/var/www/MAGE` sans réutiliser la variable ?